src/Microsoft.AspNetCore.Http.Extensions/RequestHeaders.cs

```diff
@@ -260,6 +260,23 @@ public RangeHeaderValue Range
                 Headers.Set(HeaderNames.Range, value);
             }
         }
+        
+        public Uri Referer
+        {
+            get
+            {
+                Uri uri;
+                if (Uri.TryCreate(Headers[HeaderNames.Referer], UriKind.RelativeOrAbsolute, out uri))
+                {
+                    return uri;
+                }
+                return null;
+            }
+            set
+            {
+                Headers.Set(HeaderNames.Referer, value == null ? null : UriHelper.Encode(value));
```

Comment:
Hmm, the Travis builder says this line doesn't compile. Are you missing a using?

Reply:
Oops. Yeah, it was a missing `using` statement for the `UriHelper.Encode` extension method.